framework/test/randoop_coverage.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+################################################################################
+#
+# This script generates coverage data for Randoop generated tests over the defects4j suite.
+#
+################################################################################
+# Import helper subroutines and variables, and init Defects4J
+source test.include
+init
+
+# master coverage file
+master_coverage=$TMP_DIR/coverage
+
+# Directory for Randoop test suites
+randoop_dir=$TMP_DIR/randoop
+
+# Generate tests for all projects
+# Mockito #1 and #3 don't work; problem finding classes in byte-buddy?
+# Thus the strange hack below
```

Comment:
This seems related to issue #89, which we have fixed. Can we remove this special case?